### PR TITLE
[GE] Color Picker Changes + Bug Fixes

### PR DESCRIPTION
**Pull request by msDestiny14**
(Needs testing before merging!!!!)
https://github.com/BabylonJS/ThePirateCove/issues/9
https://github.com/BabylonJS/ThePirateCove/issues/13

Would like @PatrickRyanMS to review :)
https://github.com/BabylonJS/ThePirateCove/issues/18
https://github.com/BabylonJS/ThePirateCove/issues/23

image

### CODE REVIEW

File: guiEditor/src/components/propertyTab/propertyTab.scss
Comment (RaananW) [inline guiEditor/src/components/propertyTab/propertyTab.scss:328]: sharable color? is it used somewhere else?

**Review by RaananW — APPROVED** [review]
Some conflicts and a small question. otherwise LGTM! :-)

**Comment by RaananW** [comment]
Before merging - wait for the second review

File: guiEditor/src/components/propertyTab/propertyTab.scss
Comment (msDestiny14) [reply guiEditor/src/components/propertyTab/propertyTab.scss:328]: No, In fact this might need to be black. Let me check!

File: guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx
Comment (PatrickRyanMS) [inline guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx:104]: is iconLabel = {"Sca;e"} correct? The semicolon stands out as a potential typo

**Review by PatrickRyanMS — COMMENTED** [review]
only one comment on what looks like a typo

File: guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx
Comment (msDestiny14) [reply guiEditor/src/components/propertyTab/propertyGrids/gui/commonControlPropertyGridComponent.tsx:104]: AHA. Whoops. ;)

**Comment by PatrickRyanMS** [comment]
Last thing I noticed is that the "transparency" checkerboard on the canvas is not implemented. Should I open a separate issue for that? This is the second checkbox on [https://github.com/BabylonJS/ThePirateCove/issues/23](https://github.com/BabylonJS/ThePirateCove/issues/23)

**Comment by msDestiny14** [comment]
Yes please open a separate issue.

File: guiEditor/src/components/propertyTab/propertyTabComponent.tsx
Comment (PatrickRyanMS) [inline guiEditor/src/components/propertyTab/propertyTabComponent.tsx:346]: iconLabel={"Bacground Color"} is this missing a "k" in Background Color?

**Pull request merged**
